.travis.yml

```diff
@@ -158,3 +158,4 @@ install:
 
 script:
 - tox -v
+- pip install -e .  # Validates that project can install in editable mode
```

Comment:
If you want this check, it should go to pre-commit as one of the linters there.

Reply:
I already considered this but I am afraid that pre-commit already knows to use PEP-518 and to avoid use of setup.py. Still, it could be possible to write a small in-repo custom hook that performs the packaging test. In fact it may prove to be good idea to have a generic hook for testing packaging. I will try to write and propose on to pre-commit-hooks.